db/migrate/20150930183738_migrate_content_hosts.rb

```diff
@@ -329,7 +329,7 @@ def update_hypervisor_consumer(hypervisor)
   # rubocop:disable Metrics/MethodLength
   # rubocop:disable Metrics/AbcSize
   def up
-    if User.where(:login => User::ANONYMOUS_API_ADMIN).first.nil?
+    if User.unscoped.where(:login => User::ANONYMOUS_API_ADMIN).first.nil?
```

Comment:
Not needed, but shouldn't this also be `exists?` like the other?
```suggestion
    if User.unscoped.where(:login => User::ANONYMOUS_API_ADMIN).exists?
```